### ISSUE TITLE
where is the esriAddInX file?

### ISSUE THREAD

**Issue by ThomasHaarlem** [cyclomedia/streetsmart-arcgis-pro#31]
I've downloaded the code als "streetsmart-arcgis-pro-develop.zip"
I've unzipped all the files and folders.

I expect the esrieAddinX file in this folder:
C:\...\streetsmart-arcgis-pro-develop\client-side\C#\StreetSmartArcGISPro\StreetSmartArcGISPro\AddIns
But I don't see it. Also with Windows Explorer I cannot find the esriAddinX file.

Now I'm stuck at this part of the Street Smart for ArcGIS Pro User Manual:
1) IV: Browse to "_the_" folder where "_the_" Add-In file is located. 

I hope someone can give me a direction to the solution. Thanks.

**Comment by Harm73** [comment]
I don't know if this problem still exists. It had escaped my notice. Here is only the code for building the AddIn. The esriAddinX file can be downloaded from the 'ArcGIS Marketplace'.

**Comment by ThomasHaarlem** [comment]
yeah it still exists. I didnt know to look for it at the ArcGIS Marketplace. There I see you have to pay for the plugin. Unfortunate...
